src/DRICH_geo.cpp

```diff
@@ -384,17 +388,31 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
     // cut overlaps with other sectors using "pie slice" wedges, to the extent specified
     // by `mirrorPhiw`
     Tube              pieSlice(0.01 * cm, vesselRmax2, tankLength / 2.0, -mirrorPhiw / 2.0, mirrorPhiw / 2.0);
+    Tube              pieSlice1(0.01 * cm, vesselRmax2, tankLength / 2.0, -mirrorPhiw / 2.0, (0.05*mirrorPhiw) / 2.0);
+    Tube              pieSlice2(0.01 * cm, vesselRmax2, tankLength / 2.0, (0.06*mirrorPhiw) / 2.0, mirrorPhiw / 2.0);
     IntersectionSolid mirrorSolid2(pieSlice, mirrorSolid1, mirrorPlacement);
+    IntersectionSolid mirrorSolid4(pieSlice1, mirrorSolid3, mirrorPlacement);
+    IntersectionSolid mirrorSolid5(pieSlice2, mirrorSolid3, mirrorPlacement);
```

Comment:
Perhaps it's time to come up with clearer names than using incrementing numbers, since we are now at "5".

Reply:
I don't have any good ideas, but maybe something like
- mirrorSolidInnerTile
- mirrorSolidOuterTileA
- mirrorSolidOuterTileB

Or you could put these in some STL container, such as `std::set`, so that you don't really have to give them names.